xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/NotificationConfiguration.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.notifications;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+import org.xwiki.configuration.ConfigurationSource;
+
+/**
+ * Get the configuration options concerning the Notification module.
+ * 
+ * @version $Id$
+ */
+@Component(roles = NotificationConfiguration.class)
+@Singleton
+public class NotificationConfiguration
+{
+    private static final String CONFIGURATION_PREFIX = "notification.";
+
+    @Inject
+    private ConfigurationSource configurationSource;
+
+    /**
+     * @return true if the UI for the notification module should be enabled
+     */
+    public boolean isUIEnabled()
```

Comment:
@vmassol: You're right, I did not think about the email feature since it does not exist yet.
@aubincleme: The property should be called `isEnabled()` indeed.